content/webapp/app.ts

```diff
@@ -104,56 +104,30 @@ const appPromise = nextApp
       pageVanityUrl(router, nextApp, url, prismicId, template)
     );
 
-    route('/exhibitions', '/exhibitions', router, nextApp);
     route(
       `/exhibitions/:period(${periodPaths})`,
       '/exhibitions',
       router,
       nextApp
     );
-    route('/exhibitions/:id', '/exhibition', router, nextApp);
 
-    route('/events', '/events', router, nextApp);
     route(`/events/:period(${periodPaths})`, '/events', router, nextApp);
-    route(`/events/:id(${prismicId})`, '/event', router, nextApp);
-    route(`/event-series/:id(${prismicId})`, '/event-series', router, nextApp);
-
-    route(`/stories/:contentType`, '/article-series-many', router, nextApp);
 
-    route('/articles', '/articles', router, nextApp);
-    route(`/articles/:id(${prismicId})`, '/article', router, nextApp);
-    route(`/series/:id(${prismicId})`, '/article-series', router, nextApp);
     route(`/projects/:id(${prismicId})`, '/page', router, nextApp);
 
-    route('/books', '/books', router, nextApp);
-    route(`/books/:id(${prismicId})`, '/book', router, nextApp);
-
-    route(`/places/:id(${prismicId})`, '/place', router, nextApp);
```

Comment:
This is the one route that doesn't have a replacement, because `/place` goes nowhere on the current site.